src/modules/vtol_att_control/tiltrotor.h

```diff
@@ -75,7 +75,7 @@ class Tiltrotor : public VtolType
 		param_t front_trans_dur_p2;
 	} _params_handles_tiltrotor;
 
-	enum vtol_mode {
+	enum class vtol_mode {
```

Comment:
And this one too? So this same enum is declared everywhere?

Reply:
Per type privately.